Config.lua

```diff
@@ -467,44 +484,21 @@ local spells = {
 	}
 }
 
-for i = 1, MAX_CLASSES do
-	local class = CLASS_SORT_ORDER[i]
-	local name = LOCALIZED_CLASS_NAMES_MALE[class]
-	spells.args[class] = {
-		icon = "Interface\\Icons\\ClassIcon_" .. class,
-		iconCoords = borderlessCoords,
-		name = name,
-		type = "group",
-		get = function(info)
-			return getSpellOption(info)
-		end,
-		set = function(info, val)
-			setSpellOption(info, val)
-		end,
-		args = createOptionsForClass(class)
-	}
-end
 
 local additionalSpellCategories = {
-	["TRINKET"] = "Trinket",
-	["GENERAL"] = "General"
+	["TRINKET"] = L["Trinket"],
```

Comment:
This should be INVTYPE_TRINKET instead of L["Trinket"] as it's already defined. Realistically we should probably remove L["Trinket"].

Reply:
I also found that it should be remove  L["Trinket"]
L["General Spells"] Can't find where to use it?

Reply:
L["General Spells"] should also be removed as it's no longer used.

Reply:
please removed